inc/SimpleExpressionCalculatorW3.h

```diff
@@ -0,0 +1,51 @@
+///////////////////////////////////////////////////////////////////////////////
+/**
+ * @file  SimpleExpressionCalculatorW3.h
+ * @date  Wed, 2 June 2021
+ * @brief 
+ * 
+ */
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef SIMPLEEXPRESSIONCALCULATORW3_H
+#define SIMPLEEXPRESSIONCALCULATORW3_H
+
+#include <string>
+
+#include "ICalculatorW3.h"
+
+namespace calculator 
+{
+    class SimpleExpressionCalculatorW3 : public ICalculatorW3
+    {
+        public:
+            /**
+             * @brief The left side of the equation [leftTerm] [operation] [rightTerm],  e.g. '5 + 5'
+             * 
+             * @return std::string 
+             */
+            std::string getExpression() override final;
```

Comment:
in that case, shouldn't we rename this function to "getLeftExpression"?

Reply:
The expression is the left part of the equation

![image](https://user-images.githubusercontent.com/17300617/120860342-c5ae9b00-c542-11eb-92a1-603ae9ecd320.png)